docs/content/config/advanced/tips/server-multiple-ip-addresses.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Use Cases | Binding DMS to a specific network'
+hide:
+  - toc
+---
+
+!!! warning "Advice not extensively tested"
+
+    This configuration has only been used with `network: host`, where you have direct access to the host interfaces.
```

Comment:
```suggestion
!!! warning "Advice not extensively tested"

    This configuration advice is a community contribution which has only been verified as a solution when using `network: host`, where you have direct access to the host interfaces.

    It may be applicable in other network modes if the container has control of the outbound IPs to bind to. This is not the case with bridge networks that typically bind to a private range network for containers which are bridged to a public interface via Docker.
```